scripts/deploy.sh

```
@@ -54,6 +54,33 @@ if [ "$ENV" = "prod" ] || [ "$ENV" = "all" ]; then
 fi
 
+# 컨테이너 healthcheck 대기
+wait_healthy() {
+    local compose_files="$1"
+    local service="$2"
+    local max_attempts=10
+    local interval=10
+
+    echo "==> Waiting for $service to become healthy..."
+    for i in $(seq 1 $max_attempts); do
+        status=$(docker compose $compose_files ps "$service" --format '{{.Health}}' 2>/dev/null || echo "unknown")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)